analyze

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+DEBUG_LOG_FILE_PATH=$1
+DEBUG_LOG_FILE_NAME=$(basename "$DEBUG_LOG_FILE_PATH")
+DEBUG_LOG_DIR_PATH=$(dirname "$DEBUG_LOG_FILE_PATH")
+
+shift
+
+docker pull langered/bosh-director-logs-parser
+docker run --rm -itv "$DEBUG_LOG_DIR_PATH":/usr/src/bosh-director-logs-parser/logs langered/bosh-director-logs-parser:latest "/usr/src/bosh-director-logs-parser/logs/$DEBUG_LOG_FILE_NAME" "$@"
```

Comment:
Would be nice to have an option to use the R script as volume instead of the copied version in the docker container. That way you don't have to recreate the docker image after changing the R script. As users anyhow need to have this repo checked out, maybe this could even make a good default.